distribution/docker/build.gradle

```diff
@@ -149,9 +142,6 @@ project.ext {
           // build-context sub-modules.
           duplicatesStrategy = DuplicatesStrategy.INCLUDE
           from projectDir.resolve("src/docker/config")
-          if (oss) {
-            from projectDir.resolve("src/docker/config/oss")
```

Comment:
You can remove the directory `distribution/docker/src/docker/config/oss/` as well now.